src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -7996,6 +7996,12 @@ private void ParseDeclarationModifiers(SyntaxListBuilder list)
                     mod = this.AddError(mod, ErrorCode.ERR_BadMemberFlag, mod.Text);
                 }
 
+                // check for duplicates, can only be const
```

Comment:
Comments shouldn't be used to break apart `if / else` blocks.  Please move the comment into the else block.